.github/workflows/build_impl.yml

```diff
@@ -6,13 +6,11 @@ on:
         description: Runner to run the job on
         required: true
         type: string
-        default: heavy
```

Comment:
These options are already specified everywhere, and default values make it more difficult to reason on the call side what's going on, so let's remove where it's easy